scripts/jenkins-yoctobuild-build.sh

```diff
@@ -220,25 +239,57 @@ fi
 
 
 if [ "$RUN_INTEGRATION_TESTS" = "true" ]; then
-    cd $WORKSPACE
-    # Set build dir for qemu again, BBB build might possibly have overridden
-    # this.
-    source oe-init-build-env build-qemu
-    prepare_and_set_PATH
+    # this isn't a PR test
+    if [ -z $PR_TO_TEST ]; then
+        cd $WORKSPACE
+        # Set build dir for qemu again, BBB build might possibly have overridden
+        # this.
+        source oe-init-build-env build-qemu
+        prepare_and_set_PATH
 
 
-    cd $WORKSPACE/meta-mender/meta-mender-qemu
-    cp $BUILDDIR/tmp/deploy/images/vexpress-qemu/core-image-full-cmdline-vexpress-qemu.{ext4,sdimg} .
-    cp $BUILDDIR/tmp/deploy/images/vexpress-qemu/u-boot.elf .
+        cd $WORKSPACE/meta-mender/meta-mender-qemu
+        cp $BUILDDIR/tmp/deploy/images/vexpress-qemu/core-image-full-cmdline-vexpress-qemu.{ext4,sdimg} .
+        cp $BUILDDIR/tmp/deploy/images/vexpress-qemu/u-boot.elf .
 
-    docker build -t mendersoftware/mender-client-qemu:pr --build-arg VEXPRESS_IMAGE=core-image-full-cmdline-vexpress-qemu.sdimg --build-arg UBOOT_ELF=u-boot.elf .
-    $WORKSPACE/integration/extra/release_tool.py --set-version-of mender --version pr
-    cd $WORKSPACE/integration/tests && ./run.sh
+
+        # only create a mender pr container if we build qemu
+        if [ "$BUILD_QEMU" = "true" ]; then
+          docker build -t mendersoftware/mender-client-qemu:pr --build-arg VEXPRESS_IMAGE=core-image-full-cmdline-vexpress-qemu.sdimg --build-arg UBOOT_ELF=u-boot.elf .
+          $WORKSPACE/integration/extra/release_tool.py --set-version-of mender --version pr
+        fi
```

Comment:
Still one whitespace issue here (but you can probably get rid of this indentation, see my earlier comment).